provider/k8s.go

```diff
@@ -483,7 +815,7 @@ func (k8s *K8sOfflineStore) GetPrimaryTable(id ResourceID) (PrimaryTable, error)
 }
 
 func (k8s *K8sOfflineStore) CreateResourceTable(id ResourceID, schema TableSchema) (OfflineTable, error) {
-	return nil, nil
+	return nil, fmt.Errorf("not implemented")
```

Comment:
Do we need to implement this?

Reply:
it is only for testing relational databases, since they make resource tables by creating views rather than references, the way we do it.